relays/client-substrate/src/chain.rs

```diff
@@ -72,6 +73,9 @@ pub trait RelayChain: Chain {
 	const PARACHAINS_FINALITY_PALLET_NAME: &'static str;
 }
 
+/// Substrate-based parachain from minimal relay-client point of view.
+pub trait Parachain: Chain + ParachainBase {}
```

Comment:
Is it really required? If so, can we do a blanket impl here (`impl<T: Chain + ParachainBase> Parachain for T {}`)?

Reply:
It's not required, for the moment it's just a convenience trait (we can specify only `Parachain` instead of `Chain + ParachainBase`). I think this helps. Also maybe it would be better to keep it in case in the future we need to add more properties to the `Parachain` trait on the relayer side. But I don't have strong preferences here.